### PR TITLE
Cumulus 201 Add ParsePdr and IngestGranule workflows and tests

### PR DESCRIPTION
**Pull request by abarciauskas-bgse**
**Summary:** Adds ParsePdr and IngestGranule workflows and tests

Addresses [CUMULUS-201](https://bugs.earthdata.nasa.gov/browse/CUMULUS-201)

## Changes

* Adds ParsePdr and IngestGranule workflows and tests
* Addresses eslint findings
* Adds documentation to README about deployment and test configuration

## Test Plan
- [x] Unit tests
- [x] Adhoc testing

## Todo
- [x] Deploy release of cumulus and update the @cumulus/integration-tests package for this repo. This should fix test failure on CircleCI and enable the current pending test for IngestGranule.

### CODE REVIEW

File: spec/parsePdr/ParsePdrSuccessSpec.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+const inputTemplateFilename = './spec/parsePdr/ParsePdr.input.template.json';
+const templatedInputFilename = templateFile({
+  inputTemplateFilename,
+  config: awsConfig[taskName]
+});
+const expectedParsePdrOutput = JSON.parse(fs.readFileSync('./spec/parsePdr/ParsePdr.output.json'));
+const pdrFilename = 'MOD09GQ_1granule_v3.PDR';
+
+jasmine.DEFAULT_TIMEOUT_INTERVAL = 120000;
```
Comment (laurenfrederick) [inline spec/parsePdr/ParsePdrSuccessSpec.js:18]: I think we can set this globally in `spec/run.js`. I should have moved it there.

File: spec/parsePdr/ParsePdrSuccessSpec.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    workflowExecution = await workflow.executeWorkflow(
+      awsConfig.stackName,
+      awsConfig.bucket,
+      taskName,
+      templatedInputFilename
+    );
+  });
+
+  afterAll(async () => {
+    await aws.deleteS3Object(awsConfig.bucket, `${awsConfig.stackName}/pdrs/${pdrFilename}`);
```
Comment (laurenfrederick) [inline spec/parsePdr/ParsePdrSuccessSpec.js]: Is this created by the workflow? If so, can you add a comment?

**Review by laurenfrederick — APPROVED** [review]
Some very minor changes. Otherwise, great work. This is such an exciting change and will help with debugging issues, reproducing bugs, etc.

File: package.json [hunk 3]
```diff
@@ -1,40 +1,48 @@
 {
-  "name": "cumulus-test-deploy",
+  "name": "cumulus-integration-tests",
   "version": "1.0.0",
   "description": "Cumulus Integration Test Deployment",
   "main": "index.js",
   "scripts": {
-    "test": "babel-node spec/run.js",
+    "test": "jasmine",
```
Comment (scisco) [inline package.json]: This change prevent this from working in node 6 because babel is not used

**Review by scisco — CHANGES_REQUESTED** [review]
your tests fail because you are not using the `babel-node spec/run.js` If you revert it back to how it is on the master branch, it should work.

**Review by scisco — APPROVED** [review]

**Pull request merged**